less/components/map.less

```diff
@@ -20,3 +20,25 @@
   position: relative;
   height: 100%;
 }
+
+.marker-popup {
+  padding-left: 0;
+  margin-bottom: 0;
+  list-style-type: none;
+  li {
+    &:first-child {
+      border-top: none;
```

Comment:
This doesn't seem to have any effect, I think because the `listItemBorderColorizer` used to style lists on the page has higher [CSS specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity). One gross way to fix this is by adding `!important`, but I think a better way would be by fixing #1002, which I just filed.

One option is for you to fix #1002 in this PR, and another is to simply remove the above rule and we can fix #1002 in a separate PR.

Also, personally I think it would be cool to also have `border-bottom: none` on the `:last-child`, so that the orange lines only appear *between* clubs... Applying the same styling for the clubs list below the map would be awesome too (that's actually how I originally wanted to style the clubs list, but at the time I was in a hurry and didn't want to deal with #1002, which was a prerequisite). But like you said, it's just a temporary fix--albeit one that will likely be around for a while because our priorities for the site will be elsewhere for the next several weeks at least--so it's up to you.